### ISSUE TITLE
Add Marker/Region Support

### ISSUE THREAD

**Issue by LykaiosNZ** [LykaiosNZ/reaper-osc.js#17]
Relevant messages:
```
REWIND_FORWARD_BYMARKER t/bymarker
REWIND_FORWARD_SETLOOP t/editloop
GOTO_MARKER i/marker t/marker/@
GOTO_REGION i/region t/region/@

MARKER_NAME s/marker/@/name
MARKER_NUMBER s/marker/@/number/str
MARKER_TIME f/marker/@/time
REGION_NAME s/region/@/name
REGION_NUMBER s/region/@/number/str
REGION_TIME f/region/@/time
REGION_LENGTH f/region/@/length
LAST_MARKER_NAME s/lastmarker/name
LAST_MARKER_NUMBER s/lastmarker/number/str
LAST_MARKER_TIME f/lastmarker/time
LAST_REGION_NAME s/lastregion/name
LAST_REGION_NUMBER s/lastregion/number/str
LAST_REGION_TIME f/lastregion/time
LAST_REGION_LENGTH f/lastregion/length

MARKERID_NAME s/marker_id/@/name
MARKERID_TIME f/marker_id/@/time
MARKERID_NUMBER i/marker_id/@/number
REGIONID_NAME s/region_id/@/name
REGIONID_TIME f/region_id/@/time
REGIONID_LENGTH f/region_id/@/length
REGIONID_NUMBER i/region_id/@/number
```

**Comment by MeestorX** [comment]
Do you happen to know the OSC command to delete a marker?

**Comment by LykaiosNZ** [comment]
I'm not sure if it's possible, it doesn't mention in the [pattern file](https://github.com/LykaiosNZ/reaper-osc.js/blob/main/patterns/reaper-osc-js.ReaperOSC) and finding details about exactly how the messages work it very difficult. 

It may be possible to do by setting a marker name or number to an empty value?

Also, look like the `DEVICE_MARKER_COUNT` and `DEVICE_REGION_COUNT` default to `0` in the patterrn file so you will need to update that if you want to enable Reaper to send messages about them

**Comment by MeestorX** [comment]
> I'm not sure if it's possible, it doesn't mention in the [pattern file](https://github.com/LykaiosNZ/reaper-osc.js/blob/main/patterns/reaper-osc-js.ReaperOSC) and finding details about exactly how the messages work it very difficult.
> 
> It may be possible to do by setting a marker name or number to an empty value?

Yeah, tried several different ways of blank #, blank name, etc. no joy.

> Edit: had a play with sending various messages, seems like you can hide a marker by setting its time to a negative value but it doesn't actually remove it

I was able to make a M-1 marker. ;-)

I'll ask on the Reaper forum. They are one of the least-helpful resources out there, but who knows, maybe someone will take pity on me. I'll update if I find the answer.

**Comment by MeestorX** [comment]
Looks like the closest thing is `/action/40613` "delete marker near cursor".
Not very satisfying as you'd have to move the play head to the marker then do that action.
